### DISCUSSION TITLE
git push error

### DISCUSSION THREAD

**Discussion by cuneytda** [community/community#86066] [Discussions]
### Select Topic Area

Question

### Body

Hi,
I added a Google login feature to my project. I get the following warning as a result of the git push command. What's the problem?

$ git push
Enumerating objects: 24319, done.
Counting objects: 100% (24319/24319), done.
Delta compression using up to 2 threads
Compressing objects: 100% (3376/3376), done.
error: RPC failed; HTTP 400 curl 92 HTTP/2 stream 0 was not closed cleanly: CANCEL (err 8)
send-pack: unexpected disconnect while reading sideband packet
Writing objects: 100% (24298/24298), 10.23 MiB | 191.00 KiB/s, done.
Total 24298 (delta 20882), reused 24277 (delta 20862), pack-reused 0
fatal: the remote end hung up unexpectedly
Everything up-to-date

**Comment by ghost** [comment]
Try these:

- Sometimes, a simple retry resolves transient network issues.
- Ensure you have a stable internet connection.
- Verify if your repository on the remote server is intact and accessible.
- If pushing a large number of objects, splitting the changes into smaller, more manageable commits might help.

Also check your remote repository that every setting configured for proper use.


Thank you

**Reply by cuneytda** [reply]
I will try. Thank you so much.

**Comment by cuneytda** [comment]
problem solved. Thank you.

**Reply by ghost** [reply]
Your great welcome 😁
